pkg/util/crd_defaults.go

```diff
@@ -357,12 +356,11 @@ func GetAlertDefaultValue2() *alertv1.AlertSpec {
 		AlertImageVersion: "2.1.0",
 		CfsslImageName:    "hub-cfssl",
 		CfsslImageVersion: "4.8.1",
-		BlackduckHost:     "<<HUB_HOST>>",
-		BlackduckUser:     "sysadmin",
-		BlackduckPort:     &hubPort,
 		Port:              &port,
 		StandAlone:        &standAlone,
+		PVCStorageClass:   "",
 		AlertMemory:       "512M",
 		CfsslMemory:       "640M",
+		PVCMemory:         "800M",
```

Comment:
The PVC PVCMemory needs to be more than 800M
The AlertMemory default should be 2560M and CFSSL should be 640M
CFSSL should be blackduck-cfssl:1.0.0 instead of hub-cfssl:4.8.1
Alert should be version 3.1.0

Reply:
Made these updates!